common-test/src/main/java/io/strimzi/test/k8s/BaseKubeClient.java

```diff
@@ -354,4 +355,19 @@ public String describe(String resourceType, String resourceName) {
     public String logs(String pod) {
         return Exec.exec(namespacedCommand("logs", pod)).out();
     }
+
+    @Override
+    public String searchInLog(String resourceType, String resourceName, String sinceSeconds, String... grepPattern) {
+        try {
+            return Exec.exec("bash", "-c", join(" ", namespacedCommand("logs", resourceType + "/" + resourceName, "--since=" + sinceSeconds + "s",
+                    "|", "grep", "\'" + join("\\|", grepPattern) + "\'"))).out();
```

Comment:
The reason you're having to quote the `|` is `grep` defaults to using basic REs. You could use `grep -E`, which should mean you don't need to quite the pipes. Or you could use multiple `-e` options to save having to combine the patterns into a single one.